src/ports/STM32Cube/sampleapp_main.c

```diff
@@ -0,0 +1,115 @@
+/*********************************************************************
+ *        _       _         _
+ *  _ __ | |_  _ | |  __ _ | |__   ___
+ * | '__|| __|(_)| | / _` || '_ \ / __|
+ * | |   | |_  _ | || (_| || |_) |\__ \
+ * |_|    \__|(_)|_| \__,_||_.__/ |___/
+ *
+ * www.rt-labs.com
+ * Copyright 2021 rt-labs AB, Sweden.
+ *
+ * This software is dual-licensed under GPLv3 and a commercial
+ * license. See the file LICENSE.md distributed with this software for
+ * full license information.
+ ********************************************************************/
+
+#include "sampleapp_common.h"
+#include "app_utils.h"
+#include "app_log.h"
+#include "app_gsdml.h"
+
+#include "osal_log.h"
+#include "osal.h"
+#include <pnet_api.h>
+
+#include <lwip/netif.h>
+
+#include <string.h>
+
+#define APP_DEFAULT_ETHERNET_INTERFACE "st"
+#define APP_DEFAULT_FILE_DIRECTORY     ""
+#define APP_LOG_LEVEL                  APP_LOG_LEVEL_INFO
+
+#define APP_BG_WORKER_THREAD_PRIORITY  2
+#define APP_BG_WORKER_THREAD_STACKSIZE 4096 /* bytes */
+
+/********************************** Globals ***********************************/
+
+static app_data_t * sample_app = NULL;
+static pnet_cfg_t pnet_cfg = {0};
+app_args_t app_args = {0};
+
+/****************************** Main ******************************************/
+
+int _main (void)
+{
+   int ret;
+   int32_t app_log_level = APP_LOG_LEVEL;
```

Comment:
Why is this needed? Do we plan to change log level during runtime?

Reply:
This was copied from rt-kernel/sampleapp_main.c. Will remove in both files.